zipkin-junit5/pom.xml

```diff
@@ -46,16 +46,10 @@
 
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
-      <artifactId>mockwebserver3-junit5</artifactId>
```

Comment:
not sure why this dep was used because we weren't using any jupiter types defined in this package..